### ISSUE TITLE
Site Accelerator: Accelerate CSS Background Images

### ISSUE THREAD

**Issue by benjaminwolkchen** [Automattic/jetpack#16396]
#### Describe the solution you'd like
All CSS Background Images embedded in the website, e.g Images via the Gutenberg Cover Block are served via the Jetpack CDN.

#### Describe alternatives you've considered
Manually rewriting image urls of CSS Imports to the Jetpack CDN.

**Comment by jessefriedman** [comment]
Anything I can do to increase the priority of this issue? With Gutenberg blocks using background images quite heavily, Site Accelerator isn't as powerful as it could be.

**Comment by wouterkloos** [comment]
I agree with @jessefriedman. Especially since cover images (at least on my site) are mostly above the fold they play a big part in the time it takes to render. This would be the perfect place for the superpowers of the Jetpack CDN.

**Comment by stronenv** [comment]
Plus one for support for `background-image`. In addition to the Cover and Media & Text blocks, The Group block now supports background-image. 

Please let me know if there are any further details I can provide.

**Comment by mharis** [comment]
Support for core Gutenberg blocks would be appreciated.
